dist/stat_data.py

```diff
@@ -277,7 +279,11 @@ def __init__(self, name, desc, flags=''):
         'no_aggregate,no_clear,no_scale'),
     TxnStat('txn_commit', 'transactions committed'),
     TxnStat('txn_fail_cache', 'transaction failures due to cache overflow'),
+    TxnStat('txn_not_visible_checkpoint',
```

Comment:
I'd prefer to leave them out: hopefully the two "pinned_range" stats together with "evict_failed" will give a good indication of whether this change is effective.